mreg/api/v1/tests/tests.py

```diff
@@ -869,7 +869,19 @@ def test_list(self):
 
     def test_post_must_have_both_fields_400_bad_request(self):
         ret = self.assert_post_and_400('/hinfos/', {'host': self.host.id, 'cpu': 'cpuname'})
-        self.assertEqual(ret.json(), {'os': ['This field is required.']})
+        self.assertEqual(
```

Comment:
Is... this the only place we test the format of errors? Charming! 

I'd like buy a type please.